backend_jobs/ingestion_pipeline/providers/image_provider_flickr.py

```diff
@@ -78,36 +77,34 @@ def get_url_for_min_resolution(self,  min_height, min_width, image):
         # See details at https://www.flickr.com/services/api/misc.urls.html
         url = image['url']
         num_of_underscores = url.count('_')
-        min_resolution = min(min_height, min_width)
+        max_resolution = min(min_height, min_width)
```

Comment:
max_resolution -> min_resolution

Reply:
changed it to max_resolution = **max**(min_height, min_width)
can I leave it max_resolution?